bestie/server/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	hpb "github.com/enfabrica/enkit/bestie/proto"
+	"github.com/enfabrica/enkit/lib/server"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"google.golang.org/grpc"
+)
+
+type GreeterService struct {
+}
+
+func (s *GreeterService) Greet(ctx context.Context, req *hpb.GreetRequest) (*hpb.GreetResponse, error) {
+	return &hpb.GreetResponse{
+		Greeting: fmt.Sprintf("Hello, %s!", req.GetName()),
+	}, nil
+}
+
+func main() {
+	grpcs := grpc.NewServer()
+	s := &GreeterService{}
+	hpb.RegisterGreeterServer(grpcs, s)
+
+	mux := http.NewServeMux()
+	mux.Handle("/metrics", promhttp.Handler())
```

Comment:
looks good as the first PR of a skeleton. I'd like to add:
1) Example prometheus metric (is there any existing gRPC metric exporter we can use?)
2) Logging.

3) UI, with typescript and react, using grpcweb. (probably a separate directory, separate main?)

Architecture has a BES endpoint ingestor, storing data in the database, and a UI pulling data from the database.

Reply:
Agreed; this is just enough to test Cloud Run connectivity for both HTTP and gRPC requests (and maybe dive into how we do security).

Added tasks to https://enfabrica.atlassian.net/browse/INFRA-190 for the above.